lintlizard/__init__.py

```diff
@@ -34,38 +35,72 @@ def version_command(self) -> Command:
 
 TOOLS = [
     CommandTool('flake8'),
-    CommandTool('isort', run_params=('-c', '.'), fix_params=('.',)),
-    CommandTool('mypy'),
-    CommandTool('black', run_params=('--check', '.'), fix_params=('.',)),
+    CommandTool('isort', run_params=('-c',), fix_params=tuple()),
```

Comment:
Looks like `fix_params` can only be an empty tuple or a None. Should we replace it with a boolean like `supports_fixing=True`?. Or leave it here as is in case some new tools in future need extra arguments to apply fixes?

Reply:
> Or leave it here as is in case some new tools in future need extra arguments to apply fixes?

That's what I was thinking